taxonomies/ingredients.txt

```diff
@@ -27238,6 +27233,57 @@ fr:e950
 
 # EMULSIFIERS and ADDITIVES
 
+# Lecithin is a generic term to designate any group of yellow-brownish fatty substances occurring in animal and plant tissues, 
+# which are amphiphilic – they attract both water and fatty substances (and so are both hydrophilic and lipophilic), 
+# and are used for smoothing food textures, dissolving powders (emulsifying), 
+# homogenizing liquid mixtures, and repelling sticking materials.
+
+en:lecithins
+ar:ليسيثين
+bg:Лецитин
+ca:lecitina
+cs:lecitin
+da:Lecitin
+de:Lecithine
+el:λεκιθίνη
+eo:Lecitino
+es:Lecitina
+et:Letsitiin
+eu:Lezitina
+fa:لسیتین
+fi:Lesitiini
+fr:lécithine, lécithines
+gl:Lecitina
+he:לציטין
+hr:Lecitin
+hu:Lecitin
+hy:Լեցիտիններ
+id:Lesitin
+is:Lesitín
+it:Lecitina
+ja:レシチン
+ko:레시틴
+ms:Lesitin
+nl:Lecithine
+pl:Lecytyny
+pt:lecitina
+ro:Lecitină
+ru:Лецитины
+sh:Lecitin
+sl:Lecitin
+sr:Лецитин
+su:Lésitin
+sv:Lecitin
+th:เลซิทิน
+tr:Lesitin
+uk:Лецитин
+uz:Letsinlar
+vi:Lecithin
+zh:卵磷脂
+wikidata:Q241595
```

Comment:
```suggestion
wikidata:en:Q241595
```

Reply:
for wikidata en: is not necessary. wikidata is language independent.

Reply:
It is necessary for OFF to put wikidata:en: , the property system assumes that properties have a a language. For properties that are language independent, we use :en